subsys/bluetooth/audio/bap_stream.c

```diff
@@ -63,25 +64,22 @@ void bt_audio_codec_to_iso_path(struct bt_iso_chan_path *path,
 	path->cc_len = pack_bt_codec_cc(codec, path->cc);
 }
 
-#if defined(CONFIG_BT_AUDIO_UNICAST_CLIENT) || \
-	defined(CONFIG_BT_AUDIO_BROADCAST_SOURCE) || \
-	defined(CONFIG_BT_AUDIO_BROADCAST_SINK)
+#if defined(CONFIG_BT_BAP_UNICAST_CLIENT) || defined(CONFIG_BT_BAP_BROADCAST_SOURCE) ||            \
+	defined(CONFIG_BT_BAP_BROADCAST_SINK)
 void bt_audio_codec_qos_to_iso_qos(struct bt_iso_chan_io_qos *io,
 				   const struct bt_codec_qos *codec_qos)
 {
 	io->sdu = codec_qos->sdu;
 	io->phy = codec_qos->phy;
 	io->rtn = codec_qos->rtn;
 }
-#endif /* CONFIG_BT_AUDIO_UNICAST_CLIENT ||
-	* CONFIG_BT_AUDIO_BROADCAST_SOURCE ||
-	* CONFIG_BT_AUDIO_BROADCAST_SINK
+#endif /* CONFIG_BT_BAP_UNICAST_CLIENT ||                                                          \
```

Comment:
Something looks a bid odd on the auto-formatting here

Reply:
Hmm, besides `clang-format` adding the `\` it doesn't seem odd to me.